Enterprise-CMCS/macpro-mdct-mfp
# SAR - Resubmission Question

### Description
Added resubmission field in SAR. The question field is visible:
1. If report is SAR
2. If a report is in  'In Revision'
3. If a report has been submitted more than 1 time and is being re-revised


### Related ticket(s)
https://jiraent.cms.gov/browse/CMDCT-3015

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/e9b898ae-80b1-4ea0-a5a8-13b338f1d926


---
### How to test
1. Create a WP as a state user and submit it

2. Approve WP report as an admin

3. Create a Sar with state user ( field should be hidden in general information)

4. Submit SAR

5. Log in as admin and unlock the report ( field should be visible after unlocking)

6. Log back in as state user and you ( field should be visible and cleared of any value)

7. Fill out the field and resubmit the SAR 

8. Log back in as an admin and the field should display new revision text in field

9. If you unlock the SAR again as an admin and return to the SAR as a state user the 'resubmission' field should reset to an empty field



---
### Author checklist




---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
